scripts/obtain/dotnet-install.sh

```diff
@@ -534,17 +534,18 @@ get_absolute_path() {
 copy_files_or_dirs_from_list() {
     eval $invocation
 
-    local root_path=$(remove_trailing_slash $1)
-    local out_path=$(remove_trailing_slash $2)
-    local override=$3
-    local override_switch=$(if [ "$override" = false ]; then printf -- "-n"; fi)
+    local root_path="$(remove_trailing_slash "$1")"
+    local out_path="$(remove_trailing_slash "$2")"
+    local override="$3"
+    local override_switch="$(if [ "$override" = false ]; then printf -- "-n"; fi)"
     
     cat | uniq | while read -r file_path; do
-        local path=$(remove_beginning_slash ${file_path#$root_path})
-        local target=$out_path/$path
+        local path="$(remove_beginning_slash "${file_path#$root_path}")"
+        local target="$out_path"/"$path"
         if [ "$override" = true ] || (! ([ -d "$target" ] || [ -e "$target" ])); then
-            mkdir -p $out_path/$(dirname $path)
-            cp -R $override_switch $root_path/$path $target
+            mkdir -p "$out_path"/"$(dirname "$path")"
+            say "cp -R $override_switch $root_path/$path $target"
+            cp -R "$override_switch" "$root_path"/"$path" "$target"
```

Comment:
This shouldn't be quoted: "$override_switch"

Reply:
Yes, discovered that just now